src/emulator/storage/apis/firebase.ts

```diff
@@ -517,3 +518,28 @@ function setObjectHeaders(
     res.setHeader("Content-Language", metadata.contentLanguage);
   }
 }
+
+function validPrefix(prefix: string) {
```

Comment:
nit: isValidPrefix

Reply:
This is intentionally matching naming in storage backend (for easier cross-reference). I'm open to review a CL that renames it internally for consistency though.

Reply:
I don't think the two need to move in lockstep, we can clean up the internal version too but let's try to follow the style guide here

Reply:
Sure, renamed